docs/comment_copy_guidelines.md

```diff
@@ -0,0 +1,133 @@
+# Comment Copy Guidelines
+
+This document sets guidelines for the comments produced by the analyzer. The
+[Feedback guidelines][git-feedback-guidelines] talk about _when_ to comment and
+_what_ to comment. This document talks mostly about _how_ to comment.
+
+## Working directory for contents
+
+Comments for the analyzer live in the [`exercism/website-copy`][git-website-copy]
+repository. Changes should be made via a PR on this repository, only changing
+the comments for one analyzer at a time. These will get the
+[`type/analyzer-comments` label][git-website-copy-label].
```

Comment:
It might be worth adding a section or a paragraph on the format of the comments and their mapping to the `folder/file` structure.
Could be something like this:

> Comments are saved in `*.md` files, one comment per file. A comments handle output by the analyzer should be mapped to the corresponding file structure by replacing `.` with `/`:
> - `csharp.general.convert_field_to_constant` => `automated-comments/csharp/general/convert_field_to_const.md`
> - `go.two-fer.missing_package_comment` => `automated-comments/go/two-fer/missing_package_comment.md`

Reply:
Maybe a link to the output definition of the analyzer where there is more the comments format.

Reply:
I've asked a question on [Slack about this](https://exercism-team.slack.com/archives/CFMEG1H0C/p1562020228338300) too.